includes/Features/Abilities_Explorer/assets/js/abilities-explorer.js

```diff
@@ -0,0 +1,388 @@
+/**
+ * Ability Explorer Admin JavaScript
+ *
+ * @package WordPress\AI\Features\Abilities_Explorer
+ * @since 0.1.0
+ */
+
+(function($) {
+	'use strict';
+
+	// Initialize on document ready
+	$(document).ready(function() {
+		AiAbilityExplorer.init();
+	});
+
+	/**
+	 * Main AI Ability Explorer object
+	 */
+	const AiAbilityExplorer = {
+		/**
+		 * Initialize
+		 */
+		init: function() {
+			this.initTestRunner();
+			this.initCopyButtons();
+			this.initValidation();
+		},
+
+		/**
+		 * Initialize Test Runner
+		 */
+		initTestRunner: function() {
+			const self = this;
+
+			// Invoke ability button
+			$('#ability-test-invoke').on('click', function() {
+				const abilitySlug = $(this).data('ability');
+				const input = $('#ability-test-payload').val();
+
+				self.invokeAbility(abilitySlug, input);
+			});
+
+			// Validate button
+			$('#ability-test-validate').on('click', function() {
+				self.validateInput();
+			});
+
+			// Clear result button
+			$('#ability-test-clear').on('click', function() {
+				$('#ability-test-result-container').hide();
+				$('#ability-test-result').html('');
+				$('#ability-test-validation').hide();
+			});
+
+			// Auto-format JSON on blur
+			$('#ability-test-payload').on('blur', function() {
+				self.formatJSON($(this));
+			});
+		},
+
+		/**
+		 * Initialize Copy Buttons
+		 */
+		initCopyButtons: function() {
+			const self = this;
+
+			$('.ability-copy-btn').on('click', function() {
+				const targetId = $(this).data('copy');
+				const $target = $('#' + targetId);
+
+				if ($target.length) {
+					self.copyToClipboard($target.text(), $(this));
+				}
+			});
+		},
+
+		/**
+		 * Initialize Validation
+		 */
+		initValidation: function() {
+			// Real-time JSON validation
+			$('#ability-test-payload').on('input', function() {
+				const $textarea = $(this);
+				const value = $textarea.val().trim();
+
+				// Clear previous validation styling
+				$textarea.removeClass('json-valid json-invalid');
+
+				if (value) {
+					try {
+						JSON.parse(value);
+						$textarea.addClass('json-valid');
+					} catch (e) {
+						$textarea.addClass('json-invalid');
+					}
+				}
+			});
+		},
+
+		/**
+		 * Invoke an ability via AJAX
+		 */
+		invokeAbility: function(abilitySlug, inputString) {
+			const self = this;
+			const $button = $('#ability-test-invoke');
+			const $resultContainer = $('#ability-test-result-container');
+			const $result = $('#ability-test-result');
+
+			// Validate JSON
+			let input;
+			try {
+				input = JSON.parse(inputString);
+			} catch (e) {
+				self.showValidation(false, [aiAbilityExplorer.strings.invalidJson]);
+				return;
+			}
+
+			// Show loading state
+			$button.prop('disabled', true);
+			$button.html(aiAbilityExplorer.strings.invoking + '<span class="ability-loading"></span>');
+
+			// Make AJAX request
+			$.ajax({
```

Comment:
As mentioned elsewhere, this is a perfect example of somewhere this plugin could use a custom Ability and the JavaScript client.